File: src/main/java/org/jetbrains/fortran/lang/lexer/FortranLexer.java
package org.jetbrains.fortran.lang.lexer;

import com.intellij.lexer.FlexAdapter;

import java.io.Reader;

public class FortranLexer extends FlexAdapter{
    public final static int INITIAL = _FortranLexer.YYINITIAL;

    public FortranLexer(boolean fFixedForm) {
        super(new _FortranLexer((Reader) null, fFixedForm));
    }
}

File: src/main/kotlin/org/jetbrains/fortran/lang/lexer/FortranLexer.kt
package org.jetbrains.fortran.lang.lexer

import com.intellij.lexer.FlexAdapter
import java.io.Reader

class FortranLexer(fFixedForm: Boolean) : FlexAdapter(_FortranLexer(null as Reader?, fFixedForm)) {
    companion object {
        val INITIAL = _FortranLexer.YYINITIAL
    }
}
